skate-plugin/src/main/kotlin/com/slack/sgp/intellij/SkateService.kt

```diff
@@ -18,20 +18,25 @@ package com.slack.sgp.intellij
 import com.intellij.openapi.components.service
 import com.intellij.openapi.project.Project
 import com.intellij.openapi.project.guessProjectDir
+import com.intellij.openapi.util.Disposer
 import com.intellij.openapi.vfs.VfsUtil
 import com.intellij.openapi.wm.ToolWindowAnchor
 import com.intellij.openapi.wm.ToolWindowManager
+import com.slack.sgp.intellij.ui.WhatsNewPanelFactory
 import java.util.function.Supplier
 
 interface SkateProjectService {
   fun showWhatsNewWindow()
 }
 
+/**
+ * This file's intended purpose is to pass in the changelog file from the file path into the What's
+ * New UI of the Skate Plugin
+ */
 class SkateProjectServiceImpl(private val project: Project) : SkateProjectService {
 
   override fun showWhatsNewWindow() {
     // TODO
-    //  Make the file configurable?
     //  Only show when changed
     //  Only show latest changes
     val settings = project.service<SkatePluginSettings>()
```

Comment:
Can do this in a separate PR but let's make sure we connect these pieces to only show changes since the last update

Reply:
Yes for sure, should I delete these todo comments for now?

Reply:
I'd say leave 'em until we actually connect the parser to the previously seen state